omnibus/config/software/opensearch.rb

```diff
@@ -0,0 +1,66 @@
+#
+# Copyright:: Chef Software, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+name "opensearch"
+default_version "1.1.0"
+
+dependency "server-open-jre"
+
+license "Apache-2.0"
+license_file "LICENSE.txt"
+skip_transitive_dependency_licensing true
+
+relative_path "opensearch-#{version}"
+
+version "1.1.0" do
```

Comment:
Do we need to include versions prior to 1.2.4?

Reply:
might be ok to not since this is internal opensearch? 

@jashaik Do we document the versions of opensearch that can be installed in the external opensearch configuration?

Reply:
@PrajaktaPurohit No, We missed it in our docs